stable/artifactory/templates/artifactory-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.art_name }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
```

Comment:
Add standard labels. Apply everywhere. Here's a good example: 

https://github.com/kubernetes/charts/tree/master/stable/nginx-ingress/

* `name` should be `{{ template "fullname" . }}`
* `app` should be `{{ template "name" . }}`

Adapt as in nginx-ingress example using components. You will also have to update selectors in services.